ops/production-deploy.tmpl.yaml

```diff
@@ -371,11 +405,18 @@ extraDeploy:
               {{- toYaml . | nindent 8 }}
             {{- end }}
 
+podSecurityContext:
+  runAsUser: 1001
+  runAsGroup: 101
+  fsGroup: 101
+  fsGroupChangePolicy: "OnRootMismatch"
+
 embargoRelease:
   enabled: false
 leaseRelease:
   enabled: false
 
+nameOverride: pals
 fcrepo:
```

Comment:
we weren't sure what to do about this `fcrepo:` section

Reply:
enabled should be false.

Reply:
```ruby
fcrepo:
  enabled: false
postgresql:
  enabled: false
redis:
  enabled: true
  architecture: standalone
  auth:
    password: $REDIS_PASSWORD
solr:
  enabled: false
fits:
  enabled: true
  servicePort: 8080
  subPath: /fits
```